doc/_includes/inverse.rst

```diff
@@ -74,15 +57,26 @@ The expected value of the current amplitudes at time *t* is then given by
 :math:`\hat{j}(t) = Mx(t)`, where :math:`x(t)` is a vector containing the
```

Comment:
you use M for the linear operator and the dimension. This is very confusing.

Reply:
One option is to use K for the inverse operator as it's often refered to as the imaging "kernel". Other option is to use Q for the dimension. So Q=P or Q=3P.

Reply:
I'll go with the `Q` approach because I think Matti might be partial to `M` for the inverse